lib/bot.js

```diff
@@ -65,19 +65,29 @@ class Bot {
     this.channelUsers = {};
 
     this.channelMapping = {};
+    this.webhooks = {};
 
     // Remove channel passwords from the mapping and lowercase IRC channel names
     _.forOwn(options.channelMapping, (ircChan, discordChan) => {
       this.channelMapping[discordChan] = ircChan.split(' ')[0].toLowerCase();
     });
 
+    // Extract id and token from Webhook urls
+    _.forOwn(options.webhooks, (url, chan) => {
```

Comment:
on second thought, might as well merge this down into the `connect` block and only have one `forOwn(options.webhooks, ...)`?